### ISSUE TITLE
Initcpio Module Error

### ISSUE THREAD

**Issue by MohammadHisham** [calamares/calamares#1797]
**Describe the bug**
BIOS - x86_64
Calamares Last Release- v3.2.44.1

**To Reproduce**
ok sorry for this lenthy behaviour
this requires a arch based system
git clone https://github.com/carbon-ui/Official-ISO-Profiles.git
sudo pacman -S archiso
sudo mkarchiso -v -w /work -o /out $HOME/Official-ISO-Profiles/GNOME

**Expected behavior**
initcpio(initial ram disk) must happen to be made

**Screenshots and Logs**
![Screenshot_20211001_200717](https://user-images.githubusercontent.com/74092485/135639168-8ab48484-488a-45c8-ac01-5e5d49efbe0c.png)
```=== START CALAMARES 3.2.44-1
2021-10-01 - 14:29:36 [1]: void Calamares::ModuleManager::loadModules()
    ERROR: Module "initramfs@initramfs" not found in module search paths. 
     "/usr/lib/calamares/modules" 
2021-10-01 - 14:29:36 [1]: void CalamaresApplication::initFailed(const QStringList&)
    ERROR: STARTUP: failed modules are ("initramfs@initramfs") 



=== START CALAMARES 3.2.44-1
2021-10-01 - 20:06:41 [1]: void Calamares::ViewManager::onInstallationFailed(const QString&, const QString&)
    ERROR: Installation failed: "Boost.Python error in job \"initcpiocfg\"." 
```

**Comment by demmm** [comment]
3.2.44 is not the last release, this issue was fixed in 3.2.44.1
Latest release is https://calamares.io/calamares-3.2.44.2-is-out/
https://github.com/calamares/calamares/releases

**Comment by MohammadHisham** [comment]
@demmm This Issue Exists
Even In The Latest Release
Please Reopen it

**Comment by philmmanjaro** [comment]
@MohammadHisham  please point to the `PKGBUILD` or package of Calamares you have used. Also check if you have 3.2.44.2-1 or later installed on your end.

**Comment by demmm** [comment]
@philmmanjaro it is resolved in IRC (3.2.44 was used)
